tests/meson.build

```diff
@@ -65,6 +65,8 @@ endif
 
 if build_gir and host_system == 'linux' and not meson.is_cross_build()
   foreach unit: ['introspection.py']
+	install_data(unit, install_dir: installed_test_bindir)
```

Comment:
Could you fix the indentation to use two spaces?

Reply:
Done, my bad.

Reply:
Thanks!